### ISSUE TITLE
TMC5240 support

### ISSUE THREAD

**Issue by G-Pereira** [teemuatlut/TMCStepper#289]
Is this library compatible (or planned) with TMC5240?

**Comment by Lurchiii** [comment]
Yes this new released chip would make things pretty interesting due to its own built in Eight-Point Motion Controller.
+1 Vote for implementation :)
